app/processing/advanced_processor.py

```
@@ -0,0 +1,826 @@
+        stats = {
+            "count": len(numeric_values),
+            "min": min(numeric_values),
+            "max": max(numeric_values),
+            "mean": statistics.mean(numeric_values),
+            "median": statistics.median(numeric_values)
+        }
+        # Add standard deviation if we have enough values
+        if len(numeric_values) > 1:
+            stats["std_dev"] = statistics.stdev(numeric_values)
+        return stats
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)